modules/idlgen/plugin/src/main/scala/IdlGenPlugin.scala

```diff
@@ -76,11 +85,13 @@ object IdlGenPlugin extends AutoPlugin {
 
   lazy val defaultSettings: Seq[Def.Setting[_]] = Seq(
     idlType := "(missing arg)",
+    srcGenSerializationType := "Avro",
     idlGenSourceDir := (Compile / sourceDirectory).value,
     idlGenTargetDir := (Compile / resourceManaged).value,
-    srcGenSerializationType := "Avro",
-    srcJarNames := Seq.empty,
+    srcGenSourceFromJarsDir := idlGenTargetDir.value / idlType.value,
```

Comment:
`idlGenTargetDir` would be unrelated since it's the opposite direction (source code to IDL files).. I think we might want to set it as `(Compile / resourceManaged).value`, even knowing that would be the same.

Reply:
with the `/  idlType.value` or without it?

Reply:
with is fine.